docs/platforms/flutter/features/index.mdx

```diff
@@ -23,3 +23,11 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
 - Integrations for sqflite, routing and more. For a complete list, see our [integrations list](/platforms/flutter/integrations/).
 - [Release Health](/product/releases/health/) tracks crash free users and sessions.
 - Limited support for Flutter Web.
+
+**Desktop Limitations:**
+
+Sentry has limited support for Flutter Desktop (Linx & Windows), but there are limitations compared with mobile platforms:
```

Comment:
```suggestion
Sentry supports Flutter on Linux and Windows as well, but there are limitations compared with mobile platforms:
```

Reply:
we should also clarify what support on macOS looks like

Reply:
we should also clarify what support on macOS looks like since users will expect macOS to fall under the `Desktop` category